accordion.info.yml

```diff
@@ -6,3 +6,6 @@ package: CKEditor
 version: VERSION
 dependencies:
   - editor
+libraries:
```

Comment:
works only in theme, in the module libraries attaches via hook https://www.drupal.org/node/2274843